proto/ibc/core/channel/v1/tx.proto

```diff
@@ -275,6 +275,9 @@ message MsgChannelUpgradeInit {
 
 // MsgChannelUpgradeInitResponse defines the MsgChannelUpgradeInit response type
 message MsgChannelUpgradeInitResponse {
+  option (gogoproto.equal)           = false;
```

Comment:
weird how this didn't result in any [`Equal`](https://pkg.go.dev/github.com/gogo/protobuf/plugin/equal) being removed from the diff. Are we sure they aren't disabled by default? I wonder why we initially started using it in other places.

Reply:
good point! i can't find it in the file at all. i've seen references to it in the SDK but always for setting `Equal` to true, i think for `Coin` messages. and looking into the docs about this directive, [equal does seem to be disabled by default.](https://pkg.go.dev/github.com/gogo/protobuf/plugin/equal)

Reply:
I was wondering the same. I'd be in favour of removing the equal directives set to false

Reply:
When I added this option in `MsgRegisterInterchainAccountResponse` in [this other PR](https://github.com/cosmos/ibc-go/pull/3856), I see that the getter functions get removed [here](https://github.com/cosmos/ibc-go/pull/3856/files#diff-9f4f1d3ff2840c8fe4c2a21f8907b11aad02a4c43d84d2855b0fdafe6fd7d1b9)...